pkg/cluster/service.go

```diff
@@ -126,16 +123,6 @@ func (s *ToolchainClusterService) addToolchainCluster(log logr.Logger, toolchain
 		Client:        cl,
 		ClusterStatus: &toolchainCluster.Status,
 	}
-	if cluster.Type == "" {
-		cluster.Type = Member
-	}
-	if cluster.OperatorNamespace == "" {
-		if cluster.Type == Host {
-			cluster.OperatorNamespace = defaultHostOperatorNamespace
-		} else {
-			cluster.OperatorNamespace = defaultMemberOperatorNamespace
-		}
-	}
```

Comment:
just curious - are we deprecating also the `cluster.OperatorNamespace` field ?

Reply:
nope, we just don't need to have a logic that would set the namespace label if it's not already there since it should be set when the ToolchainCluster CRs are created https://github.com/codeready-toolchain/toolchain-cicd/blob/34ea85fe21ae05c9bb1a0a2bbdc986b94c8d9e92/scripts/add-cluster.sh#L398

Reply:
Yeah, right, but only if the OperatorNamespace is not already set. Take a look at this line https://github.com/codeready-toolchain/toolchain-common/blob/d1606659ae07cb3ad6d1d998ebdcacdce783fc62/pkg/cluster/service.go#L208 
We set the OperatorNamespace based on the value of the label, and as I mentioned above, we can rely on the fact that the namespace label should be always set on the ToolchainCluster CR as part of its creation.
In other words, the OperatorNamespace should be always set, and if not then it should fail - see https://github.com/codeready-toolchain/toolchain-common/pull/359#discussion_r1504151698

Reply:
Ok got it, I missed the part where it was set from the label. Thanks for the explanation!